specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/network.json

```diff
@@ -83,7 +83,7 @@
     "CloudError": {
       "x-ms-external": true,
       "properties": {
-        "error": {
+        "Error": {
```

Comment:
@msyyc @jsntcy Please review this change. Will it impact CLI?

Reply:
@guptas14, Could you share the background why you need change `error` to `Error`?

Reply:
Hi,
I need to make these changes to fix the Swagger incorrectness identified by S360 automation.
There are several resource properties being returned by Networking RP service, which were missing in Swagger, or which did not tell default values.
For Error property, the Swagger definition has 'e' where all api's return error with 'E' in json object. This CloudError is a generic object for all error properties being returned.



S360 identified issues link:
https://msazure.visualstudio.com/Swagger%20KPIs/_workitems/edit/8187877

Reply:
Please check this report, for instance, generated by S360:
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=136094B8-8B18-40F7-A370-07AC2CCA81B0

Additional properties not allowed: Error
Json path: $.Error
#/definitions/CloudError
network.json#L83

@shenglol Could you please confirm here, regarding these Swagger incorrectness fixes required by NRP.